src/components/Modal/index.js

```diff
@@ -50,18 +50,18 @@ export default class Modal extends PureComponent {
 
   componentDidUpdate (prevProps) {
     const { show } = this.props
-
-    // a little hackery here to prevent scrolling/reset
+    const rootHtml = document.getElementsByTagName('html')[0]
 
     if (!prevProps.show && show) {
-      const scrollOffset = window.scrollY
-      document.body.style.top = `-${scrollOffset}px`
       document.body.classList.add('mc-modal__body--open')
+      rootHtml.classList.add('mc-modal__html--open')
```

Comment:
This here should only need the root.  From there, in the css we can target it like this if we need both overflows:

```
.mc-modal__body--open {
  overflow: hidden;
  body { overflow: hidden; }
}
```

Reply:
good catch